third-party-podspecs/glog.podspec

```diff
@@ -11,19 +11,33 @@ Pod::Spec.new do |spec|
   spec.summary = 'Google logging module'
   spec.authors = 'Google'
 
-  spec.prepare_command = File.read("../scripts/ios-configure-glog.sh")
+  spec.script_phase = {
+    :name => 'Configure Glog',
+    :script => File.read("../scripts/ios-configure-glog.sh"),
+    :execution_position => :before_compile
+  }
   spec.source = { :git => 'https://github.com/google/glog.git',
                   :tag => "v#{spec.version}" }
   spec.module_name = 'glog'
   spec.header_dir = 'glog'
   spec.source_files = 'src/glog/*.h',
+                      'src/glog/*.h.in',
+                      'libglog.pc.in',
                       'src/demangle.cc',
                       'src/logging.cc',
                       'src/raw_logging.cc',
                       'src/signalhandler.cc',
                       'src/symbolize.cc',
                       'src/utilities.cc',
-                      'src/vlog_is_on.cc'
+                      'src/vlog_is_on.cc',
+                      'src/config.h.in',
+                      'config.guess',
+                      'config.sub',
+                      'configure',
+                      'configure.ac',
+                      'Makefile.in',
+                      'install-sh',
```

Comment:
do we really need these files. Have you been verifying each of them?

Reply:
Yes they are individually added and verified. Basically, `configure` is required because the target architecture only becomes available in compile time. (whilst during pod installation, the architecture cannot be determined). And `install.sh`, `config.sub`, `config.guess` and `Makefile.in` are required by `configure` and its subroutines. The other are source files that are required to the compile the pod.